apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -346,6 +352,14 @@ export class PnpmOptionsConfiguration extends PackageManagerOptionsConfiguration
    */
   public readonly preventManualShrinkwrapChanges: boolean;
 
+  /**
+   * If true, then Rush will use the workspaces feature to install and link packages when invoking PNPM.
```

Comment:
If this is set to `true` but they are NOT using the PNPM package manager, we should report an error.

Another approach would be to move it under the `pnpmOptions` section.

We do want to enable this for Yarn pretty soon, however. But perhaps there will be different workspaces options for each package manager anyway?  And I'm actually not sure how we would approach this with NPM, since it doesn't support workspaces yet as far as I know.

Reply:
It is under `pnpmOptions`. I think this makes sense, but I do expect to do work to bring this over to Yarn sooner rather than later, so not sure if this should be it's home, or at the root level of the config (like pnpmVersion is). If you'd prefer it at the root, then yes, erroring out would definitely be required.

NPM likely will never be supported using this style of install, so I'm just going to leave that one alone.